css/style.css

```diff
@@ -32,13 +36,19 @@ ul {
   margin: 0;
 }
 
+a {
+  text-decoration: none;
+}
+
 /* -------------------------- HEADER ----------------------------- */
-header {
+
+.nav-bar-mobile-version {
   display: flex;
   flex-direction: column;
+  justify-content: space-between;
 }
```

Comment:
- [ ] Please prevent users to scroll down the page and see the content when the navbar is open, currently users can scroll down the page and see all the other sections. You can achieve that by setting:

```
#nav-bar-mobile-version {
  position: fixed;
  width: 100%;
  height: 100%;
}
```

Reply:
@ShoiraTa thank you very much